modules/nvidia_plugin/src/transformer/fuse_conv_biasadd_activation.cpp

```diff
@@ -121,21 +108,29 @@ struct FusedConvCallbacks {
     static std::pair<std::shared_ptr<TFusedConvolution>, std::shared_ptr<ov::Node>> parse_fusedconv_inputs(
         std::shared_ptr<ov::Node> add) {
         std::shared_ptr<TFusedConvolution> fused_conv = nullptr;
-        std::shared_ptr<ov::Node> node = nullptr;
-        node = add->input(1).get_source_output().get_node_shared_ptr();
-        fused_conv =
-            std::dynamic_pointer_cast<TFusedConvolution>(add->input(0).get_source_output().get_node_shared_ptr());
-        if (!fused_conv) {
-            node = add->input(0).get_source_output().get_node_shared_ptr();
-            fused_conv =
-                std::dynamic_pointer_cast<TFusedConvolution>(add->input(1).get_source_output().get_node_shared_ptr());
+
+        auto input0 = add->input(0).get_source_output().get_node_shared_ptr();
+        auto input1 = add->input(1).get_source_output().get_node_shared_ptr();
+
+        auto fused_conv0 = std::dynamic_pointer_cast<TFusedConvolution>(input0);
+        auto fused_conv1 = std::dynamic_pointer_cast<TFusedConvolution>(input1);
+
+        if (fused_conv0 && fused_conv1) {
+            if (get_node_id(fused_conv0) > get_node_id(fused_conv1)) {
+                return {fused_conv0, input1};
+            } else {
+                return {fused_conv1, input0};
+            }
         }
 
-        if (!fused_conv) {
-            return {nullptr, nullptr};
+        if (fused_conv0) {
```

Comment:
need to check node ids here as well to ensure that there's no dependency between fused_conv and input1